Source/AssistantV1/Assistant.swift

```diff
@@ -45,7 +47,6 @@ public class Assistant {
     public init(username: String, password: String, version: String) {
         self.authMethod = Shared.getAuthMethod(username: username, password: password)
         self.version = version
-        Shared.configureRestRequest()
```

Comment:
Out of curiosity, why are these being removed?

Reply:
This is the way I was adding the `User-Agent` header before. The `Shared.getMetadataHeaders()` method replaces this.